load_from_path segfault when var_paths files are not found

When a file that doesn't exist is passed to `load_from_path`, it causes a segfault:

```
$ python -c 'import pprint; import tfparse; pprint.pprint(tfparse.load_from_path("tf/", vars_paths=["notfound"]))'
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7ed0386c1055]

goroutine 17 [running, locked to thread]:
github.com/aquasecurity/trivy/pkg/iac/scanners/terraform/parser.(*evaluator).EvaluateAll(0x0, {0x7ed038bef3d8, 0x7ed0394603c0})
        /tmp/tmp_5_w6lmb/pkg/mod/github.com/cloud-custodian/trivy@v0.0.0-20240717135722-15bfb2bfe320/pkg/iac/scanners/terraform/parser/evaluator.go:125 +0x35
github.com/aquasecurity/trivy/pkg/iac/scanners/terraform/parser.(*Parser).EvaluateAll(0xc000454480, {0x7ed038bef3d8, 0x7ed0394603c0})
        /tmp/tmp_5_w6lmb/pkg/mod/github.com/cloud-custodian/trivy@v0.0.0-20240717135722-15bfb2bfe320/pkg/iac/scanners/terraform/parser/parser.go:292 +0x72
github.com/cloud-custodian/tfparse/gotfparse/pkg/converter.NewTerraformConverter({0xc000014f64, 0x3}, {0xc0004772f0, 0x2, 0x7ed038a580a0?})
        /tmp/tmp_5_w6lmb/src/github.com/cloud-custodian/tfparse/gotfparse/gotfparse/pkg/converter/converter.go:364 +0x187
main.Parse(0x0?, 0x0, 0x0, 0x0, 0x1, 0x7ffe48ffc3d0)
        /tmp/tmp_5_w6lmb/src/github.com/cloud-custodian/tfparse/gotfparse/gotfparse/cmd/tfparse/main.go:44 +0x4d5
Aborted (core dumped)
```

Note that this can also happen if the file exists, but the path is not relative to the root of the configuration, as 

```
$ python -c 'import pprint; import tfparse; pprint.pprint(tfparse.load_from_path("tf/", vars_paths=["tf/conf.tfvars"]))'
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7700504c1055]

goroutine 17 [running, locked to thread]:
github.com/aquasecurity/trivy/pkg/iac/scanners/terraform/parser.(*evaluator).EvaluateAll(0x0, {0x7700509ef3d8, 0x7700512603c0})
        /tmp/tmp_5_w6lmb/pkg/mod/github.com/cloud-custodian/trivy@v0.0.0-20240717135722-15bfb2bfe320/pkg/iac/scanners/terraform/parser/evaluator.go:125 +0x35
github.com/aquasecurity/trivy/pkg/iac/scanners/terraform/parser.(*Parser).EvaluateAll(0xc000533c20, {0x7700509ef3d8, 0x7700512603c0})
        /tmp/tmp_5_w6lmb/pkg/mod/github.com/cloud-custodian/trivy@v0.0.0-20240717135722-15bfb2bfe320/pkg/iac/scanners/terraform/parser/parser.go:292 +0x72
github.com/cloud-custodian/tfparse/gotfparse/pkg/converter.NewTerraformConverter({0xc000522a04, 0x3}, {0xc000502980, 0x2, 0x7700508580a0?})
        /tmp/tmp_5_w6lmb/src/github.com/cloud-custodian/tfparse/gotfparse/gotfparse/pkg/converter/converter.go:364 +0x187
main.Parse(0x0?, 0x0, 0x0, 0x0, 0x1, 0x7ffcfa7c9440)
        /tmp/tmp_5_w6lmb/src/github.com/cloud-custodian/tfparse/gotfparse/gotfparse/cmd/tfparse/main.go:44 +0x4d5
Aborted (core dumped)
```

I have the following files:

`tf/main.tf` with:

```
variable "dns_support" {
  type    = bool
}

resource "aws_vpc" "vpc" {
  enable_dns_support = var.dns_support
}
```

and  `tf/conf.tfvars` with

```
dns_support = true
```

--- a/setup.py
+++ b/setup.py
@@ -20,7 +20,7 @@ setup(
     maintainer="Cloud Custodian Project",
     maintainer_email="cloud-custodian@googlegroups.com",
     license="Apache-2.0",
-    version="0.6.10",
+    version="0.6.11",
     classifiers=[
         "Development Status :: 4 - Beta",
         "Intended Audience :: Developers",

